persistence/src/main/java/org/mskcc/cmo/metadb/persistence/MetaDbRequestRepository.java

```diff
@@ -25,4 +27,9 @@
             + "<-[:HAS_SAMPLE]-(p:Patient)"
             + "<-[:IS_ALIAS]-(pa:PatientAlias {value: $patientAliasId}) RETURN r")
     MetaDbRequest findMetadbRequestByPatientAlias(@Param("patientAliasId") String patientAliasId);
+
+    @Query("MATCH (r: Request {requestId: $reqId})"
+            + "MATCH (r)-[:HAS_METADATA]->(rl)"
```

Comment:
this is minor but would you mind adjusting the query to also specify the type of node? and also change the abbreviation to rm perhaps since it's a list of `RequestMetadata` that we are returning 

the second and third line specifically
```
MATCH (r)-[:HAS_METADATA]->(rm: RequestMetadata)
RETURN rm
```